ibm/mas_devops/roles/db2/tasks/install/main.yml

```diff
@@ -24,7 +24,10 @@
 # 3. Setup the norootsquash daemonsets for db2u pods to work with NFS backed storage
 # -----------------------------------------------------------------------------
 - include_tasks: tasks/install/setup_norootsquash.yml
-
+  when:
+    - db2_meta_storage_class is defined
+    - db2_data_storage_class is defined
+    - db2_data_storage_class == "ibmc-file-gold-gid" or db2_meta_storage_class == "ibmc-file-gold-gid"
```

Comment:
Can we change this to a matching "starts with ibmc-file-" so we catch any use of silver/bronze if it's out there as well.